subsys/bootloader/image/CMakeLists.txt

```diff
@@ -4,6 +4,16 @@
 # SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
 #
 
+function(add_overlay_config image overlay_file)
```

Comment:
Could this be put higher up? I want this type of functionality to be available on the app level(maybe it is?). So that you can set an image overlay file in the application's CMake file.

Reply:
The only other place to have it is currently `zephyr/cmake/extenions.cmake` and I prefer to keep this patch single repo for now :)